tests/integration_tests/family/test_search.py

```diff
@@ -7,6 +7,36 @@
 from tests.integration_tests.setup_db import setup_db
 
 
+def test_search_geographies(
+    client: TestClient, data_db: Session, superuser_header_token
+):
+    setup_db(data_db)
+
+    # search for where we know there is 2 families with a geopraphy
+    response = client.get(
+        "/api/v1/families/?geography=afghanistan",
+        headers=superuser_header_token,
+    )
+    assert response.status_code == status.HTTP_200_OK
+    data = response.json()
+    assert isinstance(data, list)
+
+    ids_found = set([f["import_id"] for f in data])
+    assert len(ids_found) == 2
+
+    # search for where we know there is 1 families with a geopraphy
```

Comment:
Can we parametrize this test by passing in the geography and the expected ids found instead of doing two checks within the same test, as the test code is the same for both `zimbabwe` and `afghanistan`?

Reply:
How do you feel about moving it to a table based test, which eeps the tests logic dry, but also avoids having to jump through an abstration to see what the test is doing?